public/sw.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable */
```

Comment:
I'd suggest also moving this file to be in `public/news/sw.js` and updating this line to pull it from the service-specific directory: https://github.com/bbc/simorgh/blob/latest/src/server/index.jsx#L114
This is to ensure that we keep our service-workers for different BBC services clearly separated.